### DISCUSSION TITLE
Comunity Plan editor

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#570] [Ideas]
We have been testing frepple and we realized it's a great tool and we think that really suits our needs, but [we miss some things](https://github.com/frePPLe/frepple/discussions/444#discussioncomment-2381832).
One of our principal needs is a visual plan editor as the cloud edition have.
By our understanding philosophy of open source (we are maintainers and core contributors of Tryton), we will love to contribute our 2 cents.

This issue is about creating a Community plan editor, so let me ask some questions, as the challenge is quite big.
- Can we pay frePPLe bvba to open source it?
- Is there any contribution we can help or maintain to help to open source the planner editor instead of redoing the same work for the community? (currently working to submit a PR for [es translations](https://github.com/frePPLe/frepple/issues/448))
- In case any of the above is possible, which are the steps more detailed to duplicate this functionality to core? I have seen some comments in this discussions like "we are open to contributions, submit your PR", but to work more efficiently I think it's better to detail which are the needs of the community and to do a blueprint prior to a PR.

**Comment by jdetaeye** [comment]
I'm writing the following with my frepple-as-a-commercial-product hat:

The plan editor has both a) the visual front-end component and b) planning engine changes to quickly propagate the interactive plan modifications (eg moving a subassembly production to a later date should also move the final assembly to a later date).  
This is a very powerful combination and far from trivial to implement.

We don't want to loose this key differentiator between the Enterprise Edition and the Community Edition.
You'll be on your own for redoing such an editor in the Community Edition. Don't expect any help from the frepple core team on this.

**Reply by zipus** [reply]
Thanks for your honest answer.
